### PR TITLE
add active date props to header

### PR DESCRIPTION
**Pull request by kbwo**
At first, thank you for great library.
I want new feature to display date of blue style except today, so I added active date props to header on this PR.

### CODE REVIEW

**Review by acro5piano — CHANGES_REQUESTED** [review]
@kbwo Thank you for your pull request!

1. Can you add this property to the [api documentation](https://github.com/acro5piano/react-native-big-calendar#api-documentation) ?
2. `isActiveDay` and `activeDate` is confusing. Do you mean highlighting a specific date (Date) or a specific day (0..6) ?

**Comment by kbwo** [comment]
> @kbwo Thank you for your pull request!
> 
> 1. Can you add this property to the [api documentation](https://github.com/acro5piano/react-native-big-calendar#api-documentation) ?
> 2. `isActiveDay` and `activeDate` is confusing. Do you mean highlighting a specific date (Date) or a specific day (0..6) ?

1. OK. I'll commit to update docs.
2. So sorry. I mean highlighting a specific date (Date). I'll also commit to rename the variable.

**Comment by kbwo** [comment]
@acro5piano 
I fixed it and update documentation.
Please check it.

File: src/components/CalendarHeader.tsx [hunk 1]
```diff
@@ -46,6 +48,7 @@ function _CalendarHeader<T>({
       <View style={[u['z-10'], u['w-50'], borderColor]} />
       {dateRange.map((date) => {
         const _isToday = isToday(date)
+        const _isActiveDate = activeDate ? isActiveDate(date, activeDate) : _isToday
```
Comment (acro5piano) [inline src/components/CalendarHeader.tsx:51]: There should be performance improvement: ​if `activeDate` is passed, `_isToday` will be calculated but not used.

So, the logic should be like this:

```ts
       ​const shouldHighlight = activeDate ? date.isSame(activeDate, 'date') : isToday(date)
       ​return (
```

Also, please note that

- The var name `shouldHighlight` is more clear
- I think the function `isActiveDate` is a bit redundant, writing it inline must be simpler 
- `isSame(.., 'date')` is more clear than `isSame(.., 'day')` in this case

Sorry that I couldn't find it at the first time ;)

**Review by acro5piano — CHANGES_REQUESTED** [review]
This should be the last fix! Thanks for your patient

File: src/components/CalendarHeader.tsx (hunk 1 above)
Comment (kbwo) [reply src/components/CalendarHeader.tsx:51]: Thank you for your review. I'll fix it!

File: src/components/CalendarHeader.tsx [hunk 2]
```diff
@@ -46,6 +48,7 @@ function _CalendarHeader<T>({
       <View style={[u['z-10'], u['w-50'], borderColor]} />
       {dateRange.map((date) => {
         const _isToday = isToday(date)
+        const shouldHighlight = activeDate ? date.isSame(activeDate, 'date') : _isToday
         return (
```
Comment (acro5piano) [inline src/components/CalendarHeader.tsx:52]: To reduce unnecessary function call, please do

```suggestion
      {dateRange.map((date) => {
        const shouldHighlight = activeDate ? date.isSame(activeDate, 'date') : isToday(date)
        return (
```

**Review by acro5piano — CHANGES_REQUESTED** [review]

**Comment by acro5piano** [comment]
Released on v2.3.0

**Pull request merged**